package.json

```diff
@@ -69,12 +69,22 @@
   "author": "dmf",
   "license": "MIT",
   "dependencies": {
+    "@babel/helper-builder-react-jsx": "^7.12.13",
+    "@babel/helper-builder-react-jsx-experimental": "^7.12.11",
+    "@babel/helper-regex": "^7.10.5",
+    "@babel/plugin-transform-react-jsx-self": "^7.12.13",
+    "@babel/plugin-transform-react-jsx-source": "^7.12.13",
```

Comment:
All of the dependency issues turned out to be needed because I was using Node version 15 rather than the Node 14 LTS release.  Downgrading to Node 14 rendered these package updates unnecessary.  I've removed these added dependencies in 713e339.

Reply:
Thanks, James! Normally I'd say we should all update our Node instead of asking you to downgrade, but I see the LTS release is still on Node 14, and the latest electron is actually running Node 12. So for those reasons, and in the interest of keeping our dependencies list small, I like your solution :)